packages/mongo/mongo_driver.js

```diff
@@ -135,12 +137,12 @@ var replaceTypes = function (document, atomTransformer) {
     return replacedTopLevelAtom;
 
   var ret = document;
-  _.each(document, function (val, key) {
+  Object.entries(document).forEach(function ([key, val]) {
     var valReplaced = replaceTypes(val, atomTransformer);
     if (val !== valReplaced) {
       // Lazy clone. Shallow copy.
       if (ret === document)
-        ret = _.clone(document);
+        ret = clone(document);
```

Comment:
Could the spread operator be used instead of lodash's `clone`?

Reply:
I did try it but it didn't work :shrug:

Reply:
That's weird. Can you try `Object.assign({}, document)` instead then? The less lodash the better.

Reply:
OK, cool, but I truly don't understand why. The CI is canceled - what was the error?